port_ocean/core/integrations/mixins/sync_raw.py

```diff
@@ -1,6 +1,7 @@
 import asyncio
 import inspect
 import typing
+from itertools import chain
```

Comment:
unused

Reply:
how does the linter didn't caught that 🤔

Reply:
good question i need to check that